core/amber/src/main/scala/edu/uci/ics/texera/web/resource/WorkflowWebsocketResource.scala

```diff
@@ -46,37 +48,14 @@ object WorkflowWebsocketResource {
   val sessionJobs = new mutable.HashMap[String, (WorkflowCompiler, ActorRef)]
 
   // Map[sessionId, Map[operatorId, List[ITuple]]]
-  val sessionResults = new mutable.HashMap[String, Map[String, List[ITuple]]]
+  val sessionResults = new mutable.HashMap[String, WorkflowResultService]
```

Comment:
I do not get this change, why make service the value of the hashmap?

Reply:
Maybe it's due to the term `service`, which usually indicates singleton. Maybe we can change a name?